react-native-purchases-ui/ios/PaywallViewWrapper.m

```diff
@@ -100,6 +103,47 @@ - (void)setOptions:(NSDictionary *)options {
     }
 }
 
+- (RCPresentedOfferingContext *)presentedOfferingContextFromOffering:(NSDictionary *)offering {
+    NSArray *availablePackages = offering[@"availablePackages"];
```

Comment:
@tonidero We're doing this on both iOS and Android (getting the first available package from the offering). For the `presentPaywall` method I'm passing the `presentedOfferingContext` separately from RN. Should we make these consistent with either one of the two approaches? I was not sure whether passing the full object from JS to natives would work well, but since we're already doing this with the full offering here.. Which one do you prefer? I'd say if we can put the logic in JS it's  at least certain that it behaves the same on both, happy to hear your thoughts on this.

Reply:
Ok thanks! We'll leave it as is